packages/sdk/identity/src/odis/query.ts

```diff
@@ -106,6 +106,18 @@ export function getServiceContext(contextName = 'mainnet') {
   }
 }
 
+export function signWithDEK(message: string, signer: EncryptionKeySigner) {
+  // Sign
+  const key = ec.keyFromPrivate(hexToBuffer(signer.rawKey))
+  const sig = JSON.stringify(key.sign(message).toDER())
```

Comment:
We really should not be writing a new signature generation function at this low of a level. (i.e. interacting directly with EC APIs) We should use the wallet or web3 interfaces if at all possible. Ideally, we would utilize EIP-712 for domain separation and typed hashing as well, but as long as we use a unique domain separator / salt, this is less of a concern.

As a more salient point of concern, the `message` input here is supposed to be a message hash. If  a caller passes in an encoded unhashed message instead of a hashed message the result will be an insecure signature. I would consider this a bit of a footgun, as is the case anytime an API (e.g. the `elliptic` API) provides a "signature" API that expects the client to do the hashing.